spec/controllers/passwords_controller_spec.rb

```diff
@@ -3,76 +3,80 @@
 require 'spec_helper'
 
 describe Api::V1::PasswordsController do
+  let!(:user) { create(:user, password: 'mypass123') }
+  let(:password_token) { user.send(:set_reset_password_token) }
+
   before :each do
     ActionMailer::Base.delivery_method = :test
     ActionMailer::Base.perform_deliveries = true
     ActionMailer::Base.deliveries = []
     Delayed::Worker.delay_jobs = false
     @request.env['devise.mapping'] = Devise.mappings[:user]
-    @user = FactoryGirl.create(
-      :user,
-      password: 'mypass123'
-    )
-    @password_token = @user.send :set_reset_password_token
   end
 
   context 'with valid params' do
-    it 'should success and send an email' do
-      post  :create,
-            user: {
-              email: @user.email
-            },
-            format: 'json'
-      expect(response.response_code).to be 204
-      expect(ActionMailer::Base.deliveries.count).to eq 1
+    it 'returns a successful response' do
+      post  :create, user: { email: user.email }, format: 'json'
+
+      expect(response.response_code).to be(204)
+    end
+
+    it 'sends an email' do
+      post  :create, user: { email: user.email }, format: 'json'
+
+      expect(ActionMailer::Base.deliveries.count).to eq(1)
     end
 
     # reset_password_token is harcoded to match the encryption of the one stored on the db
-    it 'should change the password' do
+    it 'changes the password' do
       put :update,
           user: {
             password: '123456789',
             password_confirmation: '123456789',
-            reset_password_token: @password_token
+            reset_password_token: password_token
           },
           format: 'json'
-      expect(response.response_code).to be 204
+
+      expect(response.response_code).to be(204)
```

Comment:
you are just asking for the code here? how about a message or something... it will be better if you can create a user and ask if the new password is setted.